nerdery-ecommerce/src/products/dto/products-pagination.object.ts

```diff
@@ -2,10 +2,12 @@ import { Field, ObjectType } from "@nestjs/graphql";
 import { Product } from "../entities/product.entity";
 import { PaginationMeta } from "src/common/pagination/pagination-meta.object";
 
+type ProductCursor = Product & { cursor: string };
```

Comment:
DELETE THIS! 🛑🛑🛑

THIS WAS ONLY A QUICK EXPLAINING FOR MARVIN

THIS SHOULD NOT BE SENT TO MASTER

Reply:
Solved:
```ts
import { Field, ObjectType } from "@nestjs/graphql";
import { Product } from "../entities/product.entity";
import { PaginationMeta } from "src/common/pagination/pagination-meta.object";

@ObjectType()
export class ProductsPagination {
    @Field(() => [Product])
    collection: Product[];

    @Field()
    meta: PaginationMeta;
}
```